components/event-block/EventBlockTextMedia.vue

```diff
@@ -197,26 +166,18 @@
       line-height: 1.1;
     }
 
-    .event-block-text-media__content {
-      flex: 1 1 50%;
-      margin-right: 3rem;
-    }
-
-    .event-block-text-media--mirrored .event-block-text-media__content {
-      margin-left: 3rem;
-      margin-right: 0;
+    .event-block-text-media__image {
+      margin-bottom: 0;
     }
 
     .event-block-text-media__image-wrapper {
-      flex: 1 1 50%;
-    }
-
-    .event-block-text-media__image-wrapper--desktop {
-      display: block;
+      grid-column-start: 2;
+      grid-row-start: 1;
+      grid-row-end: 99;
```

Comment:
Should get rid of this magic number but couldn't figure out how to make an element span to the last/end row, any ideas @BasVoorhoede ?